testData/solutions/win32.json

```diff
@@ -9,16 +9,114 @@
+                    "options\\.Case": "http://registry\\.gpii\\.net/common/case",
+                    "options\\.Filter": "http://registry\\.gpii\\.net/common/filter",
+                    "options\\.MixedCase": "http://registry\\.gpii\\.net/common/mixedCase",
+                    "options\\.Verbosity": "http://registry\\.gpii\\.net/common/verbosity",
+                    "options\\.JAWSPunctuationEnabled": "http://registry\\.gpii\\.net/common/jawsPunctuationEnabled",
+                    "options\\.SayWindowTypeFirst": "http://registry\\.gpii\\.net/common/sayWindowTypeFirst",
+                    "options\\.SayStateFirst": "http://registry\\.gpii\\.net/common/sayStateFirst",
+                    "options\\.InsertKeyMode": "http://registry\\.gpii\\.net/common/insertKeyMode",
+                    "options\\.CaretDetect": "http://registry\\.gpii\\.net/common/caretDetect",
+                    "options\\.CaretDetectTimeOut": "http://registry\\.gpii\\.net/common/caretDetectTimeOut",
+                    "options\\.CaretBlinkRate": "http://registry\\.gpii\\.net/common/caretBlinkRate",
+                    "options\\.SpeakANSIChars": "http://registry\\.gpii\\.net/common/speakANSIChars",
+                    "options\\.IndicateSelected": "http://registry\\.gpii\\.net/common/indicateSelected",
+                    "options\\.SaySelectedFirst": "http://registry\\.gpii\\.net/common/saySelectedFirst",
+                    "options\\.JAWSInsertKey": "http://registry\\.gpii\\.net/common/jawsInsertKey",
+                    "options\\.KeyboardType": "http://registry\\.gpii\\.net/common/keyboardType",
+                    "options\\.UseExtendedKeys": "http://registry\\.gpii\\.net/common/useExtendedKeys",
+                    "options\\.Acronyms": "http://registry\\.gpii\\.net/common/acronyms",
+                    "options\\.Synthesizer": "http://registry\\.gpii\\.net/common/synthesizer",
+                    "options\\.SleepMode": "http://registry\\.gpii\\.net/common/sleepMode",
+                    "options\\.SynthesizerResetFrequency": "http://registry\\.gpii\\.net/common/synthesizerResetFrequency",
+                    "options\\.SimultaneousSynthAndWave": "http://registry\\.gpii\\.net/common/simultaneousSynthAndWave",
+                    "options\\.BrailleDisplay": "http://registry\\.gpii\\.net/common/brailleDisplay",
+                    "options\\.DosScreenReader": "http://registry\\.gpii\\.net/common/dosScreenReader",
+                    "options\\.SayBlankLineCount": "http://registry\\.gpii\\.net/common/sayBlankLineCount",
+                    "options\\.UseWindowsKeyboardHook": "http://registry\\.gpii\\.net/common/useWindowsKeyboardHook",
+                    "options\\.SayCursorShapeChange": "http://registry\\.gpii\\.net/common/sayCursorShapeChange",
+                    "options\\.SayNumericDates": "http://registry\\.gpii\\.net/common/sayNumericDates",
+                    "options\\.SayDollars": "http://registry\\.gpii\\.net/common/sayDollars"
```

Comment:
For all the above settings/preferences, it's worth considering which are useable (and sensible) beyond Jaws. No matter which mappings you put in the above, it will always be possible for users to have the above settings in their NP set (eg. if you didn't have the JAWSInsertKey map of line 68, the user would still be able to store the JAWSInsertKey setting in their preferences set).

Generally, you should go through the list above and do the following:
* Consider whether the setting is sensible to have as a general preference that can be used beyond Jaws.
** If it is not, delete it
** If it is keep it and add it to this spreadsheet: https://docs.google.com/spreadsheets/d/1Cj9YTzM6usgd96XFn60TGN6bfMYTLCgWVMycoA4JhA8/edit#gid=0 along with the following information:
*** Name: Something concise and clear (which could very well be the name you're currently using). It doesn't need to be the same name that jaws uses - this is easily changed in the above transformations.
*** The value range: That is, values that would be sensible for a commonly used version of setting/preference. It doesn't have to fit the values that jaws uses exactly - the capabilities transformations are quite flexible when it comes to this.
*** Description: This will allow other people to understand what the setting means and when it's used. 
*** Default value: Again, this doesn't have to be the same as jaws, but it likely will, unless jaws has some odd choices on defaults.

I realize that the above is a bit of work, but we need these things (like value ranges and default values) specified for any new common term that is defined. Also, if you're in doubt about whether a setting is sensible to have as a common term or not, I suggest you leave it out for now - it's easy to create a new common term at a later point - and we might have a better sense of which are useful as we integrate more solutions.